backend/graminsta/post/services.py

```diff
@@ -0,0 +1,40 @@
+# -*- coding: UTF-8 -*-
+"""services.py"""
+
+from django.contrib.auth import models as auth_models
+from django.shortcuts import get_object_or_404
+from .serializers import UserSerializer
+from .serializers import FollowSerializer
+from .models import FollowRelationship
+
+
+def get_people_user_follows(username):
+    """
+    Returns JSON format users that the given user follows.
+    """
+    user = auth_models.User.objects.get(username=username)
```

Comment:
Try no to access `auth_models.User` directly. Instead, use `get_user_model()` to get the user model.

See also: https://docs.djangoproject.com/en/3.0/topics/auth/customizing/#django.contrib.auth.get_user_model